.circleci/config.yml

```diff
@@ -115,13 +115,57 @@ jobs:
             mv /tmp/.version.json discovery-provider/.version.json
             git add discovery-provider/.version.json
 
+            # Escape double quotes in commit message and replace git log's platform-specific newlines with @@DELIM@@
+            GIT_DIFF=$(git log --pretty=format:'• %an - %s [<https://github.com/AudiusProject/audius-protocol/commit/%H|%h>]' --abbrev-commit origin/release-v$OLD_VERSION..HEAD -- mediorum discovery-provider identity-service comms | sed 's/"/\"/g' | tr -d '\r' | sed ':a;N;$!ba;s/\n/@@DELIM@@/g')
             git commit -m "Bump version to $NEW_VERSION"
 
             git branch "release-v$NEW_VERSION"
             git checkout "release-v$NEW_VERSION"
 
             git push --set-upstream origin main "release-v$NEW_VERSION"
 
+            # Slack has a limit of 3000 characters per block, so we need to split the diff into multiple blocks
+            max_lines=8
+            json_content="{ \"blocks\": ["
+            json_content+="{ \"type\": \"header\", \"text\": { \"type\": \"plain_text\", \"text\": \"New Protocol Release Branch (v0.4.9)\n\" } },"
+            json_content+="{ \"type\": \"section\", \"text\": { \"type\": \"mrkdwn\", \"text\": \"*These changes will be released to SPs at 11am PST:*\n\" } },"
```

Comment:
is this supposed to be hard-coded?

Reply:
yep yep and this could all be tweaked as we see how daily deploys go

Reply:
ah the version being hardcoded was a typo. i'll do a followup